source/Tipster.cpp

```diff
@@ -480,9 +480,17 @@ Tipster::GetTipsFile()
 	if (roster->GetPreferredLanguages(&message) == B_OK) {
 		const char *language;
 
-		for (int32 i = 0; (language = message.GetString("language", i, NULL)) != NULL; i++)
+		for (int32 i = 0; (language = message.GetString("language", i, NULL)) != NULL; i++) {
```

Comment:
for (int32 i = 0; (language = message.GetString("language", i, NULL)) != NULL; i++) 
			if (getLocalTipsFile(ref, language) 
				|| getLocalTipsFile(ref, BString(language, BString(language).FindFirst("_"))))
				return ref;